index.html

```diff
@@ -8,7 +8,23 @@
     <title>Awesome Books</title>
 </head>
 <body>
+    <header class="nav-container">
+        <div class="logo">
+          <a href="">Awesome Books </a>
+        </div>
+        <nav class="menu">
+          <ul class="nav-list">
+            <li class="list">
+              <a href="" class="list">Book List</a>
+            </li>
```

Comment:
great job @krisbobo 🤝

- [ ] You may kindly display only the `book list` on the main page and remove the `add-book-form`.
![home](https://user-images.githubusercontent.com/63917725/180619795-049969a5-f2d3-4b94-adff-7e2edc66f712.PNG)